scripts/sbom_scraper.sh

```diff
@@ -53,18 +53,20 @@ then
     AUTHOR_EMAIL="$(git config user.email || true)"
     TOOL_NAME="$(git config --get remote.origin.url) $(git ls-files --full-name "$SCRIPTNAME")"
     TOOL_VERSION=$(git describe --tags)${GIT_STATUS:++}
+    SUPPLIER_NAME=github
+    SUPPLIER_URL=https://github.com
 else
     AUTHOR_NAME="$USER"
     AUTHOR_EMAIL=""
     TOOL_NAME="$SCRIPTNAME"
     TOOL_VERSION="unknown"
+    SUPPLIER_NAME=dockerhub
```

Comment:
move these to the conditional area below for docker image scraping

Reply:
done